src/App.js

```diff
@@ -1,16 +1,83 @@
 import React from "react";
-import logo from "./logo.png";
 import "./App.css";
+import axios from "axios";
+import WeatherData from "./WeatherData";
+import Forecast from "./Forecast";
 
 class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { input: "", weatherData: [], forecastData: {} };
+  }
+
+  handleClose = () => {
+    this.setState({ forecastData: {} });
+  };
+
+  handleShowForecast = (i) => {
+    axios
+      .get(
+        `https://api.openweathermap.org/data/2.5/forecast?lat=${this.state.weatherData[i].lat}&lon=${this.state.weatherData[i].lon}&units=metric&appid=ff74b95fa5c1b30eacf349b5b558101a`
```

Comment:
is the app id the API key? Please never include it in your commits. This belongs into the local .env file only. Please revise https://bootcamp.rocketacademy.co/2-full-stack/2.2-advanced-react/2.2.7-environmental-variables